ProcessesApi.Tests/V1/E2ETests/Fixtures/ProcessFixture.cs

```diff
@@ -56,7 +58,7 @@ protected virtual void Dispose(bool disposing)
             }
         }
 
-        private void createProcess(string state)
+        private void createstjProcess(string state)
```

Comment:
A bit nit-picky but do you mind updating the casing? I read this as 'createst jProcess'